packages/dx-react-grid/docs/reference/table-selection.md

```diff
@@ -65,6 +66,19 @@ row | any | A row.
 selected | boolean | Indicates whether a row is selected.
 onToggle | () => void | An event that initiates row selecting or deselecting.
 
+### TableSelection.RowProps
+
+Describes properties passed to a component that renders a cell containing a selection checkbox.
+
+Extends [Table.RowProps](table.md#tablerowprops)
+
+Field | Type | Description
+------|------|------------
+selectByRowClick | boolean | Indicates if row can be selected by click.
+highlighted | boolean | Indicates whether a row is highlighted.
```

Comment:
Maybe we should provide `selected` prop in addition to 'highlighted'? Because in case we do not highlight rows on select event `highlighted` will always be `false`.